src/pages/home-page/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import logo from '../../logo.svg';
+import './index.css';
+
+function HomePage() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Welcome to learning TypeScript
```

Comment:
Not too important but generally, the main text of the page isn't supposed to be in the header, and should be using a heading tag such as `h1`